tests/test_daemon_mount.cpp

```diff
@@ -267,3 +268,43 @@ TEST_F(TestDaemonMount, performanceMountsNotImplementedHasErrorFails)
     EXPECT_EQ(status.error_code(), grpc::StatusCode::FAILED_PRECONDITION);
     EXPECT_THAT(status.error_message(), StrEq("The native mounts feature is not implemented on this backend."));
 }
+
+TEST_F(TestDaemonMount, symlinkSourceGetsResolved)
+{
+    const auto [temp_dir, filename] = plant_instance_json(fake_json_contents(mac_addr, extra_interfaces));
```

Comment:
`_` is used later on line 277 so it'd cause a redefinition error. I have this one named over the other since the `filename` _might_ be useful, but the mock guard shouldn't be touched.